addons/website_sale/static/tests/tours/website_sale_complete_flow.js

```diff
@@ -179,7 +169,7 @@
         tourUtils.confirmOrder(),
     {
         content: "Check selected billing address is same as typed in previous step",
-        trigger: '#billing_address_row:contains(SO1 Billing Street Edited, 33):contains(SO1BillingCityEdited):contains(Afghanistan)',
+        trigger: 'span[data-oe-expression="order.partner_invoice_id"]:contains(SO1 Billing Street Edited, 33):contains(SO1BillingCityEdited):contains(Afghanistan)',
```

Comment:
nitpicking: This is not a hook I'd use as `data-oe-expression` does not feel reliable to me :/